src/jetstream/passthrough.go

```diff
@@ -593,6 +594,10 @@ func (p *portalProxy) ProxySingleRequest(c echo.Context) error {
 	go p.doRequest(&cnsiRequest, done)
 	res := <-done
 
+	// Copy custom content-length header from original response header
+	contentLength := "x-content-length"
+	c.Response().Header().Set(contentLength, res.ResponseHeader.Get(contentLength))
```

Comment:
i still couldn't get this to work, the status bar does not change until the download has finished.

realised why though, we need to pipe updates from the proxied request (ui backend --> epinio) to the client request (client --> ui backend)